.NET/Microsoft.Recognizers.Text.Number/NumberRecognizer.cs

```diff
@@ -81,7 +81,7 @@ public NumberRangeModel GetNumberRangeModel(string culture = null, bool fallback
 
         protected override void InitializeConfiguration()
         {
-            /*
+
             RegisterModel<NumberModel>(

```

Comment:
Is the build failure because tests are enabled and they are trying to validate resolution implementation? Or some other reason?

Reply:
I don't think so. I think this is because of duplicate test case and that will be resolved in next PR Today. You can see in attached in-line image. 
![MicrosoftTeams-image](https://user-images.githubusercontent.com/49878420/80677459-89ef0980-8ad6-11ea-814a-5105f8e914ca.png)

Reply:
Please don't create a new PR. You can update this one.
The repository also has a script to check for duplicate cases. I suggest running it before any PRs.

Reply:
Understood. I just updated this one only and built successfully. Moving forward we'll be validating for duplicate cases before raising the PRs.